src/App.js

```diff
@@ -17,45 +25,64 @@ const Wrapper = styled.section`
 `;
 
 const Text = styled.p`
-  color: white;
+  color: ${styles.colors.accentLight};
   font-size: 17px;
   font-family: "Courier new", monospace;
 `;
 
-const Title = styled(Text)` 
-  font-size: 20px; 
-  color: black;
-  background-color: papayawhip;
+const Title = styled(Text) ` 
+  font-size: 24px; 
+  color: ${styles.colors.background};
+  background-color: ${styles.colors.accent};
   padding: 5px;
-
 `;
 
 const Link = styled.a`
+  color: ${styles.colors.accentLight};
+  font-size: 17px;
+  font-family: "Courier new", monospace;
   text-decoration: none;
+  border-bottom: .5px dotted ${styles.colors.accentLight};
+  margin: 1em 0;
+`;
+
+const LinkImportant = styled(Link) `
+  font-weight: bold;
+  margin: 2em 0;
 `;
 
+const Icon = styled.svg`
+  fill: ${styles.colors.accentLight}
+  height: 20px;
+  margin: 1em 0;
+`
 
 
 class App extends Component {
 
   constructor(props) {
     super(props);
 
-    if(window.location.pathname == '/rsvp') {
+    if (window.location.pathname === '/rsvp') {
       window.location.href = 'https://uncodeferenceio.eventbrite.de';
     }
   }
 
   render() {
     return (
-      <Wrapper className="App">
-        {/*<Header />*/}
-        <Title> {"uncodeference.io"} </Title>
-        <Text> {"The web development unconference"} </Text>
-        <Text> {"Innsbruck · May 25th"} </Text>
-        <Link href="https://uncodeferenceio.eventbrite.de">
-          <Text> {"Link --> RSVP"} </Text>
-        </Link>
+      <Wrapper>
+        <Title>un<i>code</i>ference.io</Title>
+        <Text>The free web development <Link href="http://unconference.net/unconferencing-how-to-prepare-to-attend-an-unconference/">unconference</Link> in Austria</Text>
+        <Text><Link href="https://www.google.com/maps/place/Werkstätte+Wattens/@47.287262,11.5905719,17z/data=!3m1!4b1!4m5!3m4!1s0x479d6467027ebc8d:0x97e24643b15d2bec!8m2!3d47.2872584!4d11.5927606">Wattens, Tyrol</Link> &middot; May 25th, 2018</Text>
+        <LinkImportant href="https://uncodeferenceio.eventbrite.de"><b>RSVP</b></LinkImportant>
+        <Link href="http://confcodeofconduct.com/">Code of Conduct</Link>
```

Comment:
I also added a link to a code of conduct because we had a huge discussion at SoCraTes about this topic, what do you think of that?